tests/unit_tests/is_hdd.cpp

```diff
@@ -1,17 +1,36 @@
 #include "common/util.h"
+#include <cstdlib>
 #include <string>
 #include <gtest/gtest.h>
+#include <boost/optional/optional_io.hpp> /* required to output boost::optional in assertions */
+
+#ifndef GTEST_SKIP
+#include <iostream>
+#define SKIP_TEST(reason) do {std::cerr << "Skipping test: " << reason << std::endl; return;} while(0)
```

Comment:
I introduced this macro because `GTEST_SKIP` doesn't exist in fairly old versions of gtest (e.g. the vendored one, as well as the default installs on Debian 10 and Ubuntu 20), but should be available on every build in the near future, at which point I'll drop the macro.

Reply:
Link to the PR which introduced GTEST_SKIP: https://github.com/google/googletest/pull/1544